### ISSUE TITLE
An error occurred (InvalidParameterException) when calling the DescribeTasks operation: Tasks cannot be empty when the cluster is deploying with no tasks running

### ISSUE THREAD

**Issue by xgt001** [silinternational/ecs-deploy#63]
This happens when there is no task running in the ECS cluster during deployment. Say you have a single container service defined, with minimum health of 0, then the deployment causes the cluster to temporarily have 0 tasks running. This causes the error below to be thrown. 
 ```
ecs-deploy -c blah -n blah -i blah.dkr.ecr.us-east-1.amazonaws.com/blah/blah:95e7aab8 -t 120
Using image name: blah.dkr.ecr.us-east-1.amazonaws.com/blah/blah:95e7aab8
Current task definition: arn:aws:ecs:us-east-1:blah:task-definition/blah:2
New task definition: arn:aws:ecs:us-east-1:blah:task-definition/blah:3

An error occurred (InvalidParameterException) when calling the DescribeTasks operation: Tasks cannot be empty.
Service updated successfully, new task definition running.
```

Verbose:
```
+ RUNNING=
+ :
+ '[' '' ']'
+ sleep 10
+ i=20
+ '[' 20 -lt 120 ']'
++ /usr/local/bin/aws --output json ecs list-tasks --cluster blah --service-name blah --desired-status RUNNING
++ jq -r '.taskArns[]'
+ RUNNING_TASKS=
++ /usr/local/bin/aws --output json ecs describe-tasks --cluster blah --tasks
++ jq '.tasks[]| if .taskDefinitionArn == "arn:aws:ecs:us-east-1:blah:task-definition/blah:4" then . else empty end|.lastStatus'
++ grep -e RUNNING

An error occurred (InvalidParameterException) when calling the DescribeTasks operation: Tasks cannot be empty.
```
This does not impact the deployment though and the service will get correctly updated.

**Comment by fillup** [comment]
Hi there, this has actually been fixed but not merged into master yet. I'll get it merged to master shortly and roll a release for it.

**Comment by fillup** [comment]
The fix has been merged into master and released as version `2.5.0`

**Comment by xgt001** [comment]
Fantastic! Thanks a tonne!
